python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -475,6 +481,18 @@ def _init_tvt_set(
             ret = ItemSetDict(data)
         return ret
 
+    def _init_all_nodes_set(self) -> Union[ItemSet, ItemSetDict]:
+        if not isinstance(self._graph, CSCSamplingGraph):
```

Comment:
https://github.com/dmlc/dgl/blob/5dfaf99e3cd45f42f5e2bae787188c957fdcdbae/python/dgl/graphbolt/dataset.py#L76 The graph should be SamplingGraph, then you don't need to check the instance type anymore.

Reply:
As a matter of fact, in `dgl/tests/python/pytorch/graphbolt/impl/test_ondisk_dataset.py`, many temporary graphs, with a type of `NoneType`, are created in several testors, causing the testing to fail. So if I don't check the instance type here, it cannot pass the tests.

Reply:
Can you be specific about which test may fail? Is it possible to fix those first?

Reply:
https://docs.google.com/document/d/1MMSYGRrSzXRKYYbitIqcgHYsGgZBv-CeGPLO1Qfvop8/edit?usp=sharing